Source/Scene/ModelExperimental/PrimitiveRenderResources.js

```diff
@@ -184,16 +187,54 @@ export default function PrimitiveRenderResources(
    * @private
    */
   this.primitiveType = primitive.primitiveType;
+
+  /**
+   * The minimum and maximum values of this primitive's POSITION attribute.
+   * Used to construct the bounding sphere for the model that the primitive belongs to.
+   *
+   * @type {Object}
+   * @readonly
+   *
+   * @private
+   */
```

Comment:
leftover documentation?

Reply:
hm, I saw that on line 154 they documented `const primitive` in a similar way. but maybe it's not necessary for this variable?

Reply:
Yeah I don't think it's necessary here